packages/chat/components/chat/src/chat.ts

```diff
@@ -0,0 +1,355 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { LitElement} from 'lit';
+import { property, state } from 'lit/decorators.js';
+import { LlamaPluginAPI } from '../../../services/LlamaPlugin/index.js';
+// @ts-ignore
+import styles from './chat.scss?inline';
+
+//const { stablePrefix: c4aiPrefix } = settings;
+
+/**
+ * Input component using search typeahead api
+ */
+export default class C4AIChat extends LitElement {
+  static styles = styles;
+
+   /**
+   * Core array of message objects to display and interact with, invidual messages are currently described as:
+   * {"message":"displayed text", "origin":"can be bot or user, depending on user-defined role names"}
+   */
+  private _messages: any[] = [];
+  /**
```

Comment:
Nitpicking here but can you place a new line between all the property/state declarations here so it's easier to read?

Reply:
done, will keep it in mind in the future